tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -81,11 +83,34 @@ <h3>Profiling isn’t supported in Colab yet.</h3>
                always-float-label
                type="number"
                min=1
+               max=600000
+               auto-validate
+               pattern="[0-9]+"
+               error-message="Input is not an integer between 1 to 600000"
+               required
                value="{{_profileDuration}}"></paper-input>
+  <div hidden="{{!_shouldShowTpuOptions(_profilerServiceAddressType)}}">
+    <paper-checkbox checked="{{_profileIncludeDatasetOps}}">Trace dataset ops</paper-checkbox>
+    <paper-input label="Automatically retry N times when no trace event is collected"
+                 always-float-label
+                 type="number"
+                 min=1
+                 max=100
+                 auto-validate
+                 pattern="[0-9]+"
+                 error-message="Input is not an integer between 1 to 100"
+                 required
+                 value="{{_profileNumTracingAttempts}}"></paper-input>
```

Comment:
This option should be useful for GPU as well

Reply:
Done. Moved that to show in GPU capture profile page as well.